obal/data/roles/setup_sources/tasks/git.yml

```diff
@@ -7,7 +7,7 @@
   git:
     repo: "{{ setup_sources_git }}"
     dest: "{{ setup_sources_git_checkout_dir }}"
-    version: "{{ git_branch | default('master') }}"
+    version: "{{ git_branch | default('develop') }}"
```

Comment:
Would it be better to omit if unset? I think git then just uses whatever the remote indicates as default.

Reply:
indeed, in this case it's smarter to omit!